### PR TITLE
CP4  Fixes from danish zip tests and Missed code review items.

### PR DESCRIPTION
**Pull request by Camwyn**
These are things that had come up in the CP4 code reviews and zip tests, but did not get addressed before the code was merged.
Updates 2020 theme overrides for TEC-specific colors to be sure they carry through.
🐫 https://d.pr/i/c9PfHb

Creates a second reusable function for the nested color controls invertedNestedColorControl() so we can move to more DRY.

### CODE REVIEW

File: src/resources/postcss/components/full/_search.pcss [hunk 1]
```diff
@@ -125,6 +125,49 @@
...
+
+			&:active {
+				background-color: var(--tec-color-background-events-bar-submit-button-active);
+				color: var(--tec-color-text-events-bar-submit-button-active);
+			}
+		}
+	}
+
+	/* -----------------------------------------------------------------------------
+	 * Theme Overrides - Twenty Seventeen
```
Comment (paulmskim) [inline src/resources/postcss/components/full/_search.pcss:147]: Twenty Twenty

**Review by paulmskim — COMMENTED** [review]
Can we add a short description of what this is fixing + an artifact?

File: src/resources/postcss/widgets/full/_events-list.pcss [hunk 2]
```diff
@@ -88,11 +88,17 @@
...
 		.tribe-events-widget-events-list__view-more-link {
-			color: var(--tec-color-accent-primary);
+			color: var(--tec-color-link-accent);
 
-			&:hover,
-			&:focus {
-				color: var(--tec-color-accent-primary);
+			&:visited {
+				color: var(--tec-color-link-accent);
+			}
```
Comment (paulmskim) [inline src/resources/postcss/widgets/full/_events-list.pcss:95]: And remove the above:

```suggestion
			&,
			&:visited {
				color: var(--tec-color-link-accent);
			}
```

File: src/resources/postcss/widgets/full/_events-list.pcss [hunk 3]
```diff
@@ -128,13 +134,17 @@
 	.main_color .sidebar & {
 
 		.tribe-events-widget-events-list__view-more-link {
-			color: var(--tec-color-accent-primary);
+			color: var(--tec-color-link-accent);
+
+			&:visited {
+				color: var(--tec-color-link-accent);
+			}
```
Comment (paulmskim) [inline src/resources/postcss/widgets/full/_events-list.pcss:141]:
```suggestion
			&,
			&:visited {
				color: var(--tec-color-link-accent);
			}
```

**Review by paulmskim — CHANGES_REQUESTED** [review]
a couple small things, but looks good otherwise. also my eyes bleed from your color choices.

**Review by borkweb — APPROVED** [review]

**Pull request merged**